README.md

```diff
@@ -93,7 +94,8 @@ $ helm install --name my-release \
 ```
 
 **Note:** Configuration specific to:
-- Backend can be edited in [rudder-config.yaml](https://docs.rudderlabs.com/administrators-guide/config-parameters).
+
+- Backend can be edited in [rudder-config.yaml](https://docs.rudderlabs.com/administrators-guide/config-parameters). or in values.yaml under `backend.config.overrides`.
```

Comment:
The mentioned [doc link](https://docs.rudderlabs.com/administrators-guide/config-parameters)) is broken

Reply:
Created a[ separate issue](https://github.com/rudderlabs/rudderstack-helm/issues/60) to fix this after this merge

Reply:
We have deprecated this guide as it had outdated information. Should we just point to the [`config.yaml`](https://github.com/rudderlabs/rudder-server/blob/master/config/config.yaml) file? @gitcommitshow @gane5hvarma

Reply:
I can do that @ameypv-rudder